cmd/influx/cli/cli.go

```diff
@@ -114,6 +116,14 @@ func (c *CommandLine) Run() error {
 		c.ClientConfig.Password = os.Getenv("INFLUX_PASSWORD")
 	}
 
+	addr := fmt.Sprintf("%s:%d/%s", c.Host, c.Port, c.Path)
```

Comment:
is this going to default the port to 0 if not set, or does port always have a default value?

Reply:
it should always have a default at this point as there is a default for the cli params